src/nvim/testdir/test_eval.in

```diff
@@ -122,7 +121,19 @@ call SetReg('/', ['abc/'])
 call SetReg('/', ["abc/\n"])
 call SetReg('=', ['"abc/"'])
 call SetReg('=', ["\"abc/\n\""])
-
+$put ='{{{1 System clipboard'
+" Save and restore system clipboard.
+" If no connection to X-Server is possible, test should succeed.
+:let _clipreg = ['+', getreg('+'), getregtype('+')]
+:let _clipopt = &cb
+:let &cb='unnamedplus'
+:1y
+:AR +
+:tabdo :windo :echo "hi"
```

Comment:
Really? When I do `nvim -u NONE` I can still use the clipboard. True, Considering that we have changed the clipboard implementation so much anyway, this test for a very specific bug in earlier version of the vim implementation might arguably not be so relevant, but if we do want keep it, more or less repeat lines 296-299
```
      insert("one two")
      feed('^"+dwdw"+P')
      expect('two')
      eq({{'two'}, 'v'}, eval("g:test_clip['+']"))
```
but with a `execute(':tabdo :windo :echo "hi" ')` in front.